src/Snap/Snaplet/SqliteSimple.hs

```diff
@@ -135,8 +136,8 @@ instance HasSqlite (Handler b Sqlite) where
 -- > d <- nestSnaplet "db" db sqliteInit
 -- > count <- liftIO $ runReaderT (execute "INSERT ..." params) d
 instance (MonadBase IO m) => HasSqlite (ReaderT (Snaplet Sqlite) m) where
-    getSqliteState = asks (\sqlsnaplet -> sqlsnaplet ^# snapletValue)
-
+    getSqliteState = asks (^# snapletValue) -- asks (\sqlsnaplet -> sqlsnaplet ^. snapletValue)
+                      
```

Comment:
I prefer no trailing whitespace in this code, so if you need to edit this patch might clean that up.  No big deal if you don't, though.